openverse_catalog/dags/providers/provider_api_scripts/museum_victoria.py

```diff
@@ -1,203 +1,175 @@
+                media
             )
-    return image_data
-
-
-def _get_image_data(media):
-    image_url = None
-    height, width = None, None
-    if "large" in media.keys():
-        image_url = media.get("large").get("uri")
-        height = media.get("large").get("height")
-        width = media.get("large").get("width")
-
-    elif "medium" in media.keys():
-        image_url = media.get("medium").get("uri")
-        height = media.get("medium").get("height")
-        width = media.get("medium").get("width")
-
-    elif "small" in media.keys():
-        image_url = media.get("small").get("uri")
-        height = media.get("small").get("height")
-        width = media.get("small").get("width")
-
-    return image_url, height, width
-
-
-def _get_license_url(media):
-    license_url = None
-    license_ = media.get("licence")
-    if license_ is not None:
-        uri = license_.get("uri")
-        if "creativecommons" in uri:
-            license_url = uri
-    return license_url
-
-
-def _get_metadata(obj):
-    metadata = {}
-
-    metadata["datemodified"] = obj.get("dateModified")
-    metadata["category"] = obj.get("category")
-    metadata["description"] = obj.get("physicalDescription")
-
-    keywords = obj.get("keywords")
-    if type(keywords) == list:
-        metadata["keywords"] = ",".join(keywords)
-
-    classifications = obj.get("classifications")
-    if type(classifications) == list:
-        metadata["classifications"] = ",".join(classifications)
-
-    return metadata
+            license_info = VictoriaDataIngester._get_license_info(media)
+            if image_url is None or image_id is None or license_info is None:
+                continue
+            creator = VictoriaDataIngester._get_creator(media)
+
+            image: ImageDetails = {
+                "foreign_identifier": image_id,
+                "image_url": image_url,
+                "height": height,
+                "width": width,
+                "license_info": license_info,
+                "creator": creator,
+            }
+            images.append(image)
+        return images
+
+    def get_media_type(self, record: dict) -> str:
+        return "image"
+
+    @staticmethod
+    def _get_image_data(
+        media: Dict,
+    ) -> Tuple[str | None, int | None, int | None, int | None]:
+        height, width, filesize = None, None, None
+        media_data = {}
+        for size in ["large", "medium", "small"]:
+            if size in media:
+                media_data = media[size]
+                break
+        image_url = media_data.get("uri")
+        if image_url is not None:
+            height = media_data.get("height")
+            width = media_data.get("width")
+            filesize = media_data.get("size")
+        return image_url, height, width, filesize
+
+    @staticmethod
+    def _get_creator(media) -> str | None:
+        creators = media.get("creators", None)
```

Comment:
This can be called without a default, since the default is already `None`
```suggestion
        creators = media.get("creators")
```

Reply:
I made the same suggestion on one of stacimc's PR's 😆 Thank you!

Reply:
There's so many little things that can be cleaned up while we're in here refactoring :) It's a good opportunity!